From b1e327f292e5ab39a653c40d04d23f6727dae902 Mon Sep 17 00:00:00 2001
From: mmistika <mistikasoft@gmail.com>
Date: Thu, 14 Aug 2025 19:03:02 +0200
Subject: [PATCH 02/18] chore: remove config.h on 'clean' target

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index ff53040..7cd8d2d 100644
--- a/Makefile
+++ b/Makefile
@@ -49,7 +49,7 @@ xdg-shell-protocol.h:
 config.h:
 	cp config.def.h $@
 clean:
-	rm -f dwl *.o *-protocol.h
+	rm -f dwl *.o *-protocol.h config.h
 
 dist: clean
 	mkdir -p dwl-$(VERSION)
-- 
2.50.1

