From 022635fdf816a2c2a9dc962b6485789b25dd07d5 Mon Sep 17 00:00:00 2001
From: mmistika <mistikasoft@gmail.com>
Date: Thu, 14 Aug 2025 19:58:51 +0200
Subject: [PATCH 09/18] mod(kblayout): remove kblayout file and cmd

---
 config.def.h |  3 ---
 dwl.c        | 14 --------------
 2 files changed, 17 deletions(-)

diff --git a/config.def.h b/config.def.h
index b1e1c59..20ad174 100644
--- a/config.def.h
+++ b/config.def.h
@@ -19,9 +19,6 @@ static uint32_t colors[][3]                = {
 	[SchemeSel]  = { 0xeeeeeeff, 0x005577ff, 0x005577ff },
 	[SchemeUrg]  = { 0,          0,          0x770000ff },
 };
-/* keyboard layout change notification for status bar */
-static const char  kblayout_file[] = "/tmp/dwl-keymap";
-static const char *kblayout_cmd[]  = {"pkill", "-RTMIN+3", "someblocks", NULL};
 
 /* tagging - TAGCOUNT must be no greater than 31 */
 static char *tags[] = { "1", "2", "3", "4", "5", "6", "7", "8", "9" };
diff --git a/dwl.c b/dwl.c
index 95362c6..7a923a1 100644
--- a/dwl.c
+++ b/dwl.c
@@ -1868,7 +1868,6 @@ inputdevice(struct wl_listener *listener, void *data)
 void
 kblayout(KeyboardGroup *kb)
 {
-	FILE *f;
 	Client *c;
 	unsigned int idx = kb->wlr_group->keyboard.modifiers.group;
 
@@ -1880,19 +1879,6 @@ kblayout(KeyboardGroup *kb)
 	// Update client layout
 	if ((c = focustop(selmon)))
 		c->kblayout_idx = kblayout_idx;
-
-	// Save current layout to kblayout_file
-	if (*kblayout_file && (f = fopen(kblayout_file, "w"))) {
-		fputs(xkb_keymap_layout_get_name(kb->wlr_group->keyboard.keymap,
-				idx), f);
-		fclose(f);
-	}
-
-	// Run kblayout_cmd
-	if (kblayout_cmd[0] && fork() == 0) {
-		execvp(kblayout_cmd[0], (char *const *)kblayout_cmd);
-		die("dwl: execvp %s failed:", kblayout_cmd[0]);
-	}
 }
 
 int
-- 
2.50.1

