From 600f1cc2571d8289062a8d39720a4c775eab3719 Mon Sep 17 00:00:00 2001
From: mmistika <mistikasoft@gmail.com>
Date: Thu, 14 Aug 2025 20:42:29 +0200
Subject: [PATCH 15/18] conf: mouse and trackpad options

---
 config.def.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/config.def.h b/config.def.h
index 6a8969d..37efdb8 100644
--- a/config.def.h
+++ b/config.def.h
@@ -90,7 +90,7 @@ static const int left_handed = 0;
 static const int middle_button_emulation = 0;
 
 /* Natural scrolling */
-static const int trackpad_natural_scrolling = 0;
+static const int trackpad_natural_scrolling = 1;
 static const int mouse_natural_scrolling = 0;
 
 /* You can choose between:
@@ -122,7 +122,7 @@ LIBINPUT_CONFIG_ACCEL_PROFILE_ADAPTIVE
 static const enum libinput_config_accel_profile trackpad_accel_profile = LIBINPUT_CONFIG_ACCEL_PROFILE_ADAPTIVE;
 static const double trackpad_accel_speed = 0.0;
 static const enum libinput_config_accel_profile mouse_accel_profile = LIBINPUT_CONFIG_ACCEL_PROFILE_FLAT;
-static const double mouse_accel_speed = 0.0;
+static const double mouse_accel_speed = -0.4;
 
 /* You can choose between:
 LIBINPUT_CONFIG_TAP_MAP_LRM -- 1/2/3 finger tap maps to left/right/middle
-- 
2.50.1

