From 675c642e96721098530a21cf206b3733ca33c168 Mon Sep 17 00:00:00 2001
From: mmistika <mistikasoft@gmail.com>
Date: Sat, 28 Jun 2025 15:15:43 +0200
Subject: [PATCH] conf: brightness and volume bindings

Signed-off-by: mmistika <mistikasoft@gmail.com>
---
 config.def.h | 11 +++++++++--
 1 file changed, 9 insertions(+), 2 deletions(-)

diff --git a/config.def.h b/config.def.h
index 1671fe5..bb9fb1e 100644
--- a/config.def.h
+++ b/config.def.h
@@ -151,8 +151,15 @@ static const Key keys[] = {
 	/* Note that Shift changes certain key codes: c -> C, 2 -> at, etc. */
 	/* modifier                  key                 function        argument */
 	/* Screenshots */
-	{ MODKEY|WLR_MODIFIER_SHIFT, XKB_KEY_S,		 spawn,		 SHCMD("~/.scripts/screenshot_area.sh") },
-	{ MODKEY|WLR_MODIFIER_SHIFT, XKB_KEY_X,		 spawn,		 SHCMD("~/.scripts/screenshot_full.sh") },
+	{ MODKEY|WLR_MODIFIER_SHIFT, XKB_KEY_S,			 spawn,			 SHCMD("~/.scripts/screenshot_area.sh") },
+	{ MODKEY|WLR_MODIFIER_SHIFT, XKB_KEY_X,			 spawn,			 SHCMD("~/.scripts/screenshot_full.sh") },
+	/* Sound */
+	{ 0, XKB_KEY_XF86AudioRaiseVolume,				 spawn,			 SHCMD("wpctl set-volume @DEFAULT_AUDIO_SINK@ 0.01+") },
+	{ 0, XKB_KEY_XF86AudioLowerVolume,				 spawn,			 SHCMD("wpctl set-volume @DEFAULT_AUDIO_SINK@ 0.01-") },
+	{ 0, XKB_KEY_XF86AudioMute,						 spawn,			 SHCMD("wpctl set-mute @DEFAULT_AUDIO_SINK@ toggle") },
+	/* Brightness */
+	{ 0, XKB_KEY_XF86MonBrightnessUp,				 spawn,			 SHCMD("brightnessctl set 5%+") },
+	{ 0, XKB_KEY_XF86MonBrightnessDown,				 spawn,			 SHCMD("brightnessctl set 5%-") },
 	/* Default */
 	{ MODKEY,                    XKB_KEY_p,          spawn,          {.v = menucmd} },
 	{ MODKEY|WLR_MODIFIER_SHIFT, XKB_KEY_Return,     spawn,          {.v = termcmd} },
-- 
2.50.0

